integrationtests/wildfly-modules/pom.xml

```diff
@@ -137,6 +137,32 @@
          <artifactId>junit</artifactId>
          <scope>test</scope>
       </dependency>
+
+      <dependency>
+         <groupId>org.infinispan</groupId>
+         <artifactId>infinispan-query</artifactId>
+          <classifier>tests</classifier>
+         <scope>test</scope>
+      </dependency>
+
+      <dependency>
+         <groupId>org.elasticsearch.plugin</groupId>
+         <artifactId>delete-by-query</artifactId>
+         <scope>test</scope>
+      </dependency>
+
+      <dependency>
+         <groupId>org.elasticsearch</groupId>
+         <artifactId>elasticsearch</artifactId>
+         <scope>test</scope>
+      </dependency>
+
+      <dependency>
+         <groupId>log4j</groupId>
+         <artifactId>log4j</artifactId>
```

Comment:
Elasticsearch needs log4j, (note this dep is test scope), does it cause any issues?